src/Components/BookContainer/BookContainer.js

```diff
@@ -0,0 +1,73 @@
+import React, { Component } from 'react';
+import BookCard from '../../BookCard/BookCard';
+import { connect } from 'react-redux';
+import { getNYTList, addToRead } from '../../actions';
+import { getBooks } from '../../apiCalls.js';
+import './bookContainer.css';
+
+class BookContainer extends Component {
```

Comment:
We will need to remember to go through and clean up any components that are classes, but could be functional once we have a clearer picture of the Redux effect on the code